.gitignore

```diff
@@ -1,3 +1,6 @@
 .venv
 __pycache__
-log.txt
+*.txt
+*.dbk
```

Comment:
I think we will push them to repo after all fixes, at least it's needed for see what changed in docbook when we will update the sources.